#42 Removed Mockito from NetworkTest

carlosmiranda opened the pull request:
#42: Removed Mockito from `NetworkTest`. Added implementations of `push()` and `pull()` in `Remote.Fake` that can be used for testing purposes.

carlosmiranda committed:
#42 Added Remote.Fake, replace Mockito in NetworkTest

carlosmiranda committed:
Merge branch 'master' of https://github.com/zold-io/java-api into 42

carlosmiranda committed:
#42 Fixed Checkstyle error

0crat commented:
Job [#64](https://github.com/zold-io/java-api/issues/64) is now in scope, role is `REV`

<!-- https://www.0crat.com/footprint/CAZUREFND/2200417806, version: 0.29.6, hash: ${buildNumber} -->

carlosmiranda committed:
Merge branch 'master' of https://github.com/zold-io/java-api into 42

carlosmiranda committed:
#42 CR comments

carlosmiranda commented:
@victornoel please check new commit addressing your comments. Also added `TaxesTest`, which I missed due to the fact that it seems to be a recent change.

victornoel commented:
@carlosmiranda personally I like it better like that yes, also not that I am not the reviewer :)

carlosmiranda commented:
@victornoel doh. This is not even assigned yet, you are working for free. :P

victornoel commented:
@carlosmiranda just defending my interests by ensuring my future usage of those classes is a joy :D

carlosmiranda commented:
@victornoel nevertheless it should be avoided, you are basically robbing both yourself and eventual assignee of the job. It's up to the architect to maintain joyful class usage.

victornoel commented:
@carlosmiranda it's true, thanks for the reminder

carlosmiranda commented:
@llorllale are there no reviewers available? This has become a long running task

carlosmiranda commented:
@llorllale this is more than a week old now. Can you assign a `REV` or do the honor yourself?

llorllale commented:
@carlosmiranda it's a funding issue - let me try to resolve it as quickly as possible
